recipes/centos_spring_boot/Dockerfile

```diff
@@ -16,7 +16,12 @@ ARG JUNIT_VERSION=4.12
 ENV SPRING_BOOT_GROUP=org.springframework.boot
 
 COPY install_spring_boot_dependencies.sh /tmp/
+COPY ./run.sh ${HOME}/
 RUN sudo chown user:user /tmp/install_spring_boot_dependencies.sh && \
     sudo chmod a+x /tmp/install_spring_boot_dependencies.sh && \
     scl enable rh-maven33 /tmp/install_spring_boot_dependencies.sh && \
-    sudo rm -f /tmp/install_spring_boot_dependencies.sh
+    sudo rm -f /tmp/install_spring_boot_dependencies.sh && \
+    sudo chown user:user ${HOME}/run.sh && \
+    sudo chmod +x ${HOME}/run.sh
+
+CMD ["sh","-c","${HOME}/run.sh"]
```

Comment:
AFAIK the run instructions need to have infinite loop
Maybe you want entrypoint instead of command ?

also there is a missing newline

Reply:
I would always avoid ENTRYPOINT as that takes the user to customize anything later by passing parameters - let me see if I can add the infinite looping there